### DISCUSSION TITLE
How to remove/update geo search map markers manually

### DISCUSSION THREAD

**Discussion by GilbertHG** [algolia/instantsearch#5819] [Q&A]
I have a case where I should remove some of geo search map markers manually. And the items that should I have to remove they come from api so I couldn't simply just use function `transformItems` . I've tried to remove using google map function "marker.setMap(null)" ref: https://developers.google.com/maps/documentation/javascript/examples/marker-remove, but it didn't work. Is there any way to remove the markers manually?  Or do you have any way to manually update the geo search result?

**Comment by Haroenv** [comment]
Can you explain why you can't use transformItems? Do you have a sandbox or otherwise code sample of what you have and what you want to achieve? It might be interesting to use connectGeoSearch and write the map yourself if you want to stray far from the builtin capabilities
